File: LsHcore/src/main/pl/edu/osp/tiles/ShallowWater.java
package pl.edu.osp.tiles;

import pl.edu.osp.objects.Plankton;

public class ShallowWater extends Water {
	final private byte aType = 63;
	final protected byte fertility = 100;
	
	public ShallowWater() {
		staticObject = Plankton.get(fertility);
	}
}

File: LsHCore/src/main/scala/eu/brosbit/tiles/ShallowWater.scala
package eu.brosbit.tiles
import eu.brosbit.immovable.*

object ShallowWater extends TileObj:
  override val level: Byte = 0
  override val fertility: Byte = 10
  override val name: String = "shallow water"
  override val shortName: String = "l"

class ShallowWater extends Tile:
  override val aType = ShallowWater
  var imObjOpt:Option[ImmovableObject] = None
